docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
 
 .. todo::
 
-   Format as table with measure, measure definition, data sources and
-   their uses.
+   Update mortality-related data sources within existing format.
+
+Once a cause model structure is specified, data is needed to inform its states
+and transitions. For our purposes, cause models generally have the following
+data needs:
+
+#. The probability that a simulant will start the simulation in a given state
+   within the cause model.
+#. The probability that a simulant will transition to a new state within the
+   cause model in a given time-step.
+#. The disability weight for each state in the cause model
+#. The probability that a simulant in a given cause model state will die in a
+   given time-step.
```

Comment:
The biggest point this section needs to make is that our models use (approximately) instantaneous, individual-based probabilities to make decisions, but all of our data sources are period- or point-based population estimates.  That's the context for every subsection below.  

I think it's worth framing that we, in general, take population level estimates and use them as our best-guess predictors for individual-level estimates.  We then add heterogeneity to those guesses by adding more complexity to the model.  Your point 4 here has made this last step implicit.  We want to know, generally, what is the probability that a simulant will die in a given time step.  The data we actually need  for our model is how does being in a particular state in a cause model change the probability of death.

Reply:
Which we inform by starting with a population level estimate (ACMR), deleting out the population level effect of the cause (CSMR) and adding back in our best guess at the additional mortality incurred by having the condition described by particular states in the cause model (EMR).

Reply:
I added a discussion of this before the table. Let me know if you think it could still use more detail!